api/AmbientLightSensorReading.json

```diff
@@ -0,0 +1,107 @@
+{
+  "api": {
+    "AmbientLightSensorReading": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AmbientLightSensorReading",
+        "support": {
+          "webview_android": {
+            "version_added": "54"
+          },
+          "chrome": {
+            "version_added": "54"
+          },
+          "chrome_android": {
+            "version_added": "54"
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
```

Comment:
I'm not sure this is actually standards track. Doesn't seem to be in [the draft linked from the MDN page](https://www.w3.org/TR/ambient-light/), at least. Maybe this and the value below should be `false` instead?

Reply:
Yeah, this whole interface is obsolete, and was part of the old spec only. See https://www.w3.org/TR/2016/WD-ambient-light-20160830/#ambient-light-sensor-reading-interface